Zend/Optimizer/optimize_func_calls.c

```diff
@@ -272,6 +281,7 @@ void zend_optimize_func_calls(zend_op_array *op_array, zend_optimizer_ctx *ctx)
 							opline->opcode = ZEND_FETCH_STATIC_PROP_R;
 						}
 					}
+					MAKE_NOP(call_stack[call - 1].last_check_func_arg_opline);
```

Comment:
I'm not sure if this is the right place for ``MAKE_NOP``.
I would move it into ``ZEND_SEND_FUNC_ARG``.
```dif
		case ZEND_SEND_FUNC_ARG:
				/* Don't transform SEND_FUNC_ARG if any FETCH opcodes weren't transformed. */
				if (call_stack[call - 1].last_check_func_arg_opline == NULL) {
					if (opline->op2_type == IS_CONST) {
						call_stack[call - 1].try_inline = 0;
					}
					break;
+				} else {
+                  MAKE_NOP(call_stack[call - 1].last_check_func_arg_opline);
				}
				call_stack[call - 1].last_check_func_arg_opline = NULL;
				ZEND_FALLTHROUGH;
			case ZEND_SEND_VAR_EX:
```